data-migration/migrate-sharding-mysql-tidb-less-tb.md

```diff
@@ -0,0 +1,239 @@
+  host: "${host}" # For example: 172.16.10.81
+  user: "root"
+  password: "${password}" # Plaintext passwords are supported but not recommended. It is recommended that you use dmctl encrypt to encrypt plaintext passwords.
+  port: 3306
+```
+
+Run the following command in a terminal. Use `tiup dmctl` to load the data source configuration into the DM cluster:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+tiup dmctl --master-addr ${advertise-addr} operate-source create source1.yaml
+```
+
+The parameters are described as follows.
+
+|Parameter      | Description |
+|-              |-            |
+|--master-addr         | {advertise-addr} of any DM-master node in the cluster that dmctl connects to. For example: 172.16.10.71:8261|
+|operate-source create | Load data sources to the DM cluster. |
+
+Repeat the above steps until all data sources are added to the DM cluster.
+
+## Step 2: Configure the migration task
+
+Create a file named `task1.yaml` and writes the following content to it:
+
+{{< copyable "shell-regular" >}}
+
+```yaml
+name: "shard_merge"
+# Task mode. You can set it to the following:
+# - full: Performs full data migration
+# - incremental: Performs binlog real-time replication
+# - all: Performs both full data and binlog replication
+task-mode: all
+# Required for the MySQL shards. By default, the "pessimistic" mode is used.
+# If you have a deep understanding of the principles and usage limitations of the optimistic mode, you can also use the "optimistic" mode.
+# For more information, see [Merge and Migrate Data from Sharded Tables](https://docs.pingcap.com/tidb-data-migration/stable/feature-shard-merge)
+shard-mode: "pessimistic"
+meta-schema: "dm_meta"                        # A schema will be created in the downstream database to store the metadata
+ignore-checking-items: ["auto_increment_ID"]  # In this example, there are auto-incremental primary keys upstream, so you do not need to check this item.
+
+target-database:
+  host: "${host}"                             # For example: 192.168.0.1
+  port: 4000
+  user: "root"
+  password: "${password}"                     # Plaintext passwords are supported but not recommended. It is recommended that you use dmctl encrypt to encrypt plaintext passwords.
+
+mysql-instances:
+  -
+    source-id: "mysql-01"                                    # ID of the data source, which is source-id in source1.yaml
+    route-rules: ["sale-route-rule"]                         # Table route rules applied to the data source
+    filter-rules: ["store-filter-rule", "sale-filter-rule"]  # Binlog event filter rules applied to the data source
+    block-allow-list:  "log-bak-ignored"                     # Block & Allow Lists rules applied to the data source
+  -
+    source-id: "mysql-02"
+    route-rules: ["sale-route-rule"]
+    filter-rules: ["store-filter-rule", "sale-filter-rule"]
+    block-allow-list:  "log-bak-ignored"
+
+# Configurations for merging MySQL shards
+routes:
+  sale-route-rule:
```

Comment:
Better put a comment here will be better, describing what does this rule do.  (Also in Chinese version)